test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -39,126 +41,93 @@ const clusterLabel = "kcp.dev/cluster"
 
 func TestNamespaceScheduler(t *testing.T) {
 	const serverName = "main"
-	framework.RunParallel(t, "validate namespace scheduling", func(t framework.TestingTInterface, servers map[string]framework.RunningServer, artifactDir, dataDir string) {
-		ctx := context.Background()
-		if deadline, ok := t.Deadline(); ok {
-			withDeadline, cancel := context.WithDeadline(ctx, deadline)
-			t.Cleanup(cancel)
-			ctx = withDeadline
-		}
+	f := framework.NewKCPFixture(
+		framework.KcpConfig{
+			Name: serverName,
+			Args: []string{
+				"--install-cluster-controller",
+				"--install-workspace-scheduler",
+				"--install-namespace-scheduler",
+				"--discovery-poll-interval=2s",
+			},
+		},
+	)
+	defer f.SetUp(t)()
+
+	ctx := context.Background()
+	if deadline, ok := t.Deadline(); ok {
+		withDeadline, cancel := context.WithDeadline(ctx, deadline)
+		t.Cleanup(cancel)
+		ctx = withDeadline
+	}
 
-		if len(servers) != 1 {
-			t.Errorf("incorrect number of servers: %d", len(servers))
-			return
-		}
-		server := servers[serverName]
-		cfg, err := server.Config()
-		if err != nil {
-			t.Error(err)
-			return
-		}
+	require.Equalf(t, len(f.Servers), 1, "incorrect number of servers")
```

Comment:
We've got a couple competing ideas about what to do here:

 - I use the Go standard library
 - you use testify
 - @davidfestal used some assertion library

Maybe let's find a common path and not simply introduce extra ones...?

Reply:
FWIW, my personal vote is the Go standard library.

I find `require.Equal` is too easy to misuse, swapping `expected` and `actual` (as here).

Reply:
I'm using the same library as @davidfestal - it's testify in both cases.  And testify is a superset of what the go standard library does. Anyone that wants to laboriously check error codes and raise assertions is free to do so, arguably at the expense of readability.

Reply:
But we should agree as a community on a standard